client/my-sites/plan-features-2023-grid/index.tsx

```diff
@@ -476,19 +477,35 @@ export class PlanFeatures2023Grid extends Component<
 					) }
 					<header className={ headerClasses }>
 						{ isBusinessPlan( planName ) && (
-							<div className="plan-features-2023-grid__plan-logo">
-								<img src={ cloudLogo } alt="Cloud logo" />{ ' ' }
-							</div>
+							<Plans2023Tooltip
+								text={ translate(
+									'WP Cloud gives you the tools you need to add scalable, highly available, extremely fast WordPress hosting.'
+								) }
+							>
+								<div className="plan-features-2023-grid__plan-logo">
+									<img src={ cloudLogo } alt="Cloud logo" />{ ' ' }
```

Comment:
I know it's been this way, but I'm wondering if the alt text should be "WP Cloud logo" instead of just Cloud logo 🧐

Reply:
Fixed it! 🙂